presto-tests/src/test/java/com/facebook/presto/tests/TestServerInfoResource.java

```diff
@@ -47,10 +47,14 @@
 
     private ListeningExecutorService executor;
 
+    private DistributedQueryRunner queryRunner;
+
     @BeforeClass
     public void setUp()
+            throws Exception
     {
         executor = MoreExecutors.listeningDecorator(newCachedThreadPool());
+        queryRunner = createQueryRunner(TINY_SESSION, ImmutableMap.of());
```

Comment:
you need to explicitly call `close` method in `@AfterClass` for the cleanup.
Also look read about Java's try with resources to understand how the current code works.

Reply:
Yeah. Added close().